contracts/utils/StandardizedTokenDeployer.sol

```diff
@@ -14,21 +14,18 @@ import { StandardizedTokenProxy } from '../proxies/StandardizedTokenProxy.sol';
  */
 contract StandardizedTokenDeployer is IStandardizedTokenDeployer {
     Create3Deployer public immutable deployer;
-    address public immutable implementationMintBurnAddress;
-    address public immutable implementationLockUnlockAddress;
+    address public immutable implementationAddress;
 
     /**
      * @notice Constructor for the StandardizedTokenDeployer contract
      * @param deployer_ Address of the Create3Deployer contract
-     * @param implementationLockUnlockAddress_ Address of the StandardizedTokenLockUnlock contract
-     * @param implementationMintBurnAddress_ Address of the StandardizedTokenMintBurn contract
+     * @param implementationAddress_ Address of the StandardizedToken contract
      */
-    constructor(address deployer_, address implementationLockUnlockAddress_, address implementationMintBurnAddress_) {
-        if (deployer_ == address(0) || implementationLockUnlockAddress_ == address(0) || implementationMintBurnAddress_ == address(0))
+    constructor(address deployer_, address implementationAddress_) {
+        if (deployer_ == address(0) || implementationAddress_ == address(0))
             revert AddressZero();
         deployer = Create3Deployer(deployer_);
```

Comment:
We should probably add an `ICreate3Deployer` interface in gmp-sdk instead of importing the actual contract. @deanamiel